backend/src/bin/jobs/cleanUp.ts

```diff
@@ -0,0 +1,40 @@
+import { CrowdJob } from '../../types/jobTypes'
+import SequelizeRepository from '../../database/repositories/sequelizeRepository'
+import IntegrationRunRepository from '../../database/repositories/integrationRunRepository'
+import { createServiceChildLogger } from '../../utils/logging'
+import IncomingWebhookRepository from '../../database/repositories/incomingWebhookRepository'
+
+const MAX_MONTHS_TO_KEEP = 3
```

Comment:
Just wondering if keeping 1 or 2 months would be enough

Reply:
We do the same for `integrationRuns` - if they are processed successfully we delete them after 3 months.

The problem that I'm seeing is this one:
if we see a problem with the data or in the integration service we could trigger processing of an already processed stream or `incomingWebhooks` entry to solve that. This `MAX_MONTHS_TO_KEEP` is how long of a buffer do we want here?

We could also have a different value for deleting `integrationRuns` than for `incomingWebhooks`.

Reply:
Sounds good. We can keep it as is.